Detections/Syslog/ssh_NewlyInternetExposed.yaml

```diff
@@ -0,0 +1,57 @@
+﻿id: 4915c713-ab38-432e-800b-8e2d46933de6
+name: ssh newly internet-exposed endpoints
+description: endpoints with a history of sign-ins only from private IP addresses are accessed from a public IP address.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Syslog
+    dataTypes:
+      - Syslog
+queryFrequency: 24h
```

Comment:
Might make more sense to use 1d, just from a standardization perspective in Detections.  Not required.

Reply:
done